### DISCUSSION TITLE
option to disable the autocompletion for commands (colon commands)

### DISCUSSION THREAD

**Discussion by DirectXMan12** [helix-editor/helix#10409] [Ideas]
it'd be really nice to be able to disable the autocompletion box that appears when I type `:` -- it's nice for learning, but I tend to `:w` a lot, and it's *super* distracting -- it pulls my eyes away from whatever I'm looking at.

**Comment by aGuyLearning** [comment]
Could you also close this issue?
I am looking to contribute, this seems solved

**Comment by askreet** [comment]
@aGuyLearning Is this solved? I only see an option for the LSP completion and auto-info (which shows the next-key guide for e.g. 'm', not the command fuzzy-finder).
